src/platform/forms-system/src/js/definitions/directDeposit.jsx

```diff
@@ -114,23 +100,11 @@ const uiSchema = ({ affectedBenefits, unaffectedBenefits, optionalFields }) => {
     },
   };
 
-  if (!optionalFields.declineDirectDeposit) {
-    // We're not using declineDirectDeposit; Remove the entry from ui:order so
-    // it doesn't error out
-    const i = ui['ui:order'].indexOf('declineDirectDeposit');
-    if (i !== -1) ui['ui:order'].splice(i, 1);
-  }
```

Comment:
Why is this being removed?

Reply:
Related to removing `ui:order` in general